.github/workflows/deploy_seed.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy Seed
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Deploy to GCP VM
+      env:
+        PRIVATE_KEY: ${{ secrets.GCP_SSH_PRIVATE_KEY }}
+        HOST: ${{ secrets.GCP_VM_HOST }}
+        USER: ${{ secrets.GCP_VM_USER }}
+      run: |
+        echo "$PRIVATE_KEY" > private_key.pem
+        chmod 600 private_key.pem
+        ssh -o StrictHostKeyChecking=no -i private_key.pem $USER@$HOST "cd /root/rooch && git pull origin main && cargo build --release && mv ./target/release/rooch /opt/rooch/bin/ && sudo systemctl restart rooch"
```

Comment:
Do we use docker to deploy? Do not need to build --release again.

Reply:
ok, will change to docker deploy